src/contexts/app.ts

```diff
@@ -177,6 +182,18 @@ class App extends AsyncEventEmitter {
     }
   }
 
+  get backend() {
+    if (!VariablesString.exists('backend')) {
+      return HAQQ_BACKEND;
+    }
+
+    return VariablesString.get('backend') as string;
```

Comment:
Can we shorthand here?
```suggestion
    return VariablesString.get('backend') || HAQQ_BACKEND;
```